yarn.lock

```diff
@@ -155,10 +155,10 @@
   dependencies:
```

Comment:
since I've just updated packages on master, can you please override you yarn.lock with the one that is in master then launch yarn install and commit again ? (sorry : )

Reply:
Fixed in 2082a14